app/resources/forms.py

```diff
@@ -0,0 +1,39 @@
+from flask.ext.wtf import Form
+from wtforms.fields import (
+    StringField,
+    IntegerField,
+    SubmitField
+)
+from wtforms.validators import InputRequired, Length
+
+
+class ResourceForm(Form):
+    autocomplete = StringField('Enter the address')
+
+    name = StringField('Name', validators=[
+        InputRequired(),
+        Length(1, 64)
+    ])
+    description = StringField('Description', validators=[
+        InputRequired(),
+    ])
+    website = StringField('Website')
```

Comment:
^ This validator requires there to be a `http://` in front of the URL, which we may not want.